src/Utils/findTsConfig.ts

```diff
@@ -0,0 +1,22 @@
+import fs from "fs";
+import path from "path";
+
+/**
+ * Find the tsconfig.json file in the current working directory or any parent directory.
+ * This is the behavior from tsc: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html
+ *
+ * @param directory - The directory to start searching from. Defaults to the current working directory.
+ * @param filename - The filename of the tsconfig.json file. Defaults to "tsconfig.json".
+ * @returns The path to the tsconfig.json file or undefined if no such file was found.
+ */
+export function findTsConfig(directory: string = process.cwd(), filename = "tsconfig.json"): string | undefined {
```

Comment:
Is there any chance that we can import this logic from tsc rather than implementing it? If not, then this is good.